perf/lua/1mops_write.lua

```diff
@@ -10,6 +10,7 @@ local fiber = require('fiber')
 local math = require('math')
 local json = require('json')
 local fio = require('fio')
+local tarantool = require('tarantool')
 
 -- XXX: This benchmark should be able to work standalone without
 -- any provided libraries or set LUA_PATH. Add stubs for that
```

Comment:
typo in commit "perf: update the 1mops_write test to support memcs": s/pref/perf/